recipes/obspy/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set version = "1.0.2" %}
+{% set sha256 = "37d563e36a44a5309b2f53b394d7391d49c8a62b0e8d0f0d13f54ea4e2943230" %}
+# {% set md5 = "b1432aa11b6d244c4fac61737c0c9d95" %}
```

Comment:
I can, of course. I thought I'd leave them in for future reference when we'll come to the next release cycle, with checking release candidate tarballs in our feedstock..